programmingBasics.js

```diff
@@ -0,0 +1,18 @@
+const moviesTitles = [
+    "The Shawshank Redemption", 
+    "The Godfather",
+    "The Dark Knight",
+    "12 Angry Men",
+    "The Lord Of The Rings",
+    "Pulp Fiction",
+    "The Good, the Bad and the Ugly", 
+    "Forrest Gump"
+];
+
+// Question1
+moviesTitles.forEach(title => console.log(title));
+
+// Question2
+moviesTitles.forEach(title => {
+    console.log(title.split(" ").length);
+});
```

Comment:
Well done, you have solved the exercise precisely! To be exact, you did not use a literal `for` loop, but I am sure you would have been able to. The word count is also correct, maybe it would have been easier to understand the output if you wrote the title in front of the word count. ✔️